kythe/go/platform/tools/extraction/extractrepo/extractrepo.go

```diff
@@ -35,25 +35,32 @@ import (
 )
 
 var (
-	repoURI    = flag.String("remote_repo", "", "URI of the repository containing the project to be extracted")
-	repoPath   = flag.String("local_repo", "", "Existing local copy of the repository containing the project to be extracted")
+	repoURI    = flag.String("remote", "", "URI of the repository containing the project to be extracted")
+	repoPath   = flag.String("local", "", "Existing local directory copy of the repository containing the project to be extracted")
 	outputPath = flag.String("output", "", "Path for output kindex file")
 	configPath = flag.String("config", "", "Path for the JSON extraction configuration file")
 	timeout    = flag.Duration("timeout", 2*time.Minute, "Timeout for extraction")
 )
 
 func init() {
 	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, `Usage: %s -remote_repo <repo_uri> -output <path> -config [config_file_path]
+		fmt.Fprintf(os.Stderr, `Usage: %s -remote <repo_uri> -output <path> -config [config_file_path]
 
 This tool extracts compilation records from a repository utilizing a JSON
 extraction configuration encoding a kythe.proto.ExtractionConfiguration.
-This configuration is specified either via the -config flag, or else within
+
+The originating repo to extract is specified either by -remote <repo_uri> and
```

Comment:
Completely optional aesthetic nit: `<repo-url>` (feel free to ignore this, though; I just like hyphens better 🤪)